packages/typography/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { readFileSync } from "fs";
+import * as opentype from "opentype.js";
+
+const baselineHeight = 0.5;
+
+// Define a function to calculate the nudges based on font metrics
+function calculateNudges(fontPath: string, cssLineHeight: number): void {
+	// Load and parse the font
+	const fontBuffer = readFileSync(fontPath);
+	const font = opentype.parse(fontBuffer.buffer);
+
+	// Check if the font loaded correctly
+	if (!font) {
+		console.error("Could not parse the font.");
+		return;
+	}
+
+	// Extract required font metrics
+	const ascender = font.ascender;
+	const descender = font.descender;
+	const unitsPerEm = font.unitsPerEm;
+
+	console.log("ascender", ascender);
+	console.log("descender", descender);
+	console.log("unitsPerEm", unitsPerEm);
+
+	// Calculations
+	const naturalLineHeight = ascender - descender;
+	const lineHeightScale = naturalLineHeight / unitsPerEm;
+	const ascenderScale = ascender / unitsPerEm;
+	console.log("naturalLineHeight", naturalLineHeight);
+	console.log("lineHeightScale", lineHeightScale);
+	console.log("ascenderScale", ascenderScale);
+
+	const baselinePos = (cssLineHeight - lineHeightScale) / 2 + ascenderScale;
+
+	const topNudge = baselineHeight - (baselinePos % baselineHeight);
+
+	console.log("baselinePos", baselinePos);
+	console.log("topNudge", topNudge);
+	// TODO bottom nudge
+}
+
+// Read font path from command line arguments and call the function
+const fontPath = process.argv[2];
+const cssLineHeight = process.argv[3] || 1.2;
```

Comment:
the 1.2 reads like a magic number, could you move it to a constant at top of file and use something like this

```suggestion
const cssLineHeight = process.arvg[3] || 1.2
```